docs/source/getting_started.rst

```diff
@@ -1,66 +1,29 @@
-Getting Started
-===============
+Getting Started - Install
```

Comment:
These hyphens everywhere aren't necessarily the best. They should be em dashes or even better colons

Reply:
Switched to colons.